Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json

```diff
@@ -0,0 +1,22 @@
+{
+    "description": "",
+    "feed": false,
+    "id": "classifier-mapper-incoming-Cybersixgill-Actionable-Alerts",
+    "mapping": {
+        "Cybersixgill Actionable Alerts": {
+            "dontMapEventToLabels": true,
+            "internalMapping": {
+                "Suspicious domain": {
+                    "simple": "additional_info.tables.[0].suspicious_domain"
+                },
+                "Triggered domain": {
+                    "simple": "additional_info.tables.[0].triggered_domain"
```

Comment:
Please change to additional_info.tables.triggered_domain

Reply:
@syed-loginsoft - I thought we discussed the _Triggered domain_ will be a list of domains.